bin/before-all-tests.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 set -e # exiting on error
 
-docker ps # making sure docker is installed, running, and the user has permission
+docker ps > /dev/null # making sure docker is installed, running, and the user has permission
```

Comment:
if docker ps goes to dev null here then the comment should be changed

Reply:
@michaelkernaghan changed to what? 🤔 

@AlirezaHaghshenas ideas?

Reply:
making sure docker is not running

Reply:
but that's what the comment is saying, and more. It's not inaccurate

Reply:
you mean to say "making sure docker is running"?